src/microbe_stage/Microbe.Contact.cs

```diff
@@ -804,6 +804,9 @@ private IEnumerable<FloatingChunk> OnKilled()
             }
         }
 
+        // Give bonus glucose at third the amount of our compounds bag capacity
```

Comment:
Maybe can be put inside the method or as a method documentation?

Reply:
I decided to remove that comment as it was incorrect (as the multiplier was one *fourth* and not a third). So that comment got outdated even before the PR was finished, so I decided we are better off without a comment that is such a high risk for being incorrect.

Reply:
Shouldn't this be removed then (as this is still shown as an addition in the diff)?

Reply:
Oh right, I didn't realize until you pointed out. I've now removed this line.